src/frontend/src/features/layouts/components/thread-view/index.tsx

```diff
@@ -74,6 +81,10 @@ export const ThreadView = () => {
     const trashedMessageIds = messages?.results?.filter((m) => m.is_trashed).map((m) => m.id) || [];
     const isThreadTrashed = trashedMessageIds.length === messages?.results?.length;
 
+    // Get AI Configuration
+    const { data: configData } = useConfigRetrieve();
+    const config = configData?.data as ConfigWithAI | undefined;
+    const isAIEnabled = config?.AI_ENABLED === true && config?.AI_FEATURE_SUMMARY_ENABLED === true;
```

Comment:
You could put that check in a utils to prevent to repeat this logic across the app.


```suggestion
    // Get AI Configuration
    const { data: configData } = useConfigRetrieve();
    const isAIEnabled = !!configData?.data?.["AI_ENABLED"];

```